source/resolve-conflicts.ts

```diff
@@ -1,5 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference -- TODO: Is any other way supported?
 /// <reference types="codemirror" />
```

Comment:
Not if it's global, but you could try getting them to add an export too and import `codemirror` instead.